### PR TITLE
Add instructions for Kubernetes Job

### LINKED ISSUE #1111: Add docs explaining how to run Tern on Kubernetes
**Description**
On further investigation, Tern can run on Kubernetes as of release 2.9.1, but requires volume mounts. Therefore, we will just add some documentation to the README explaining how to do that with minikube.

**To Do**
Add a "how-to" document in the docs folder that explains how to run Tern using minikube. Also add a reference to this doc in the README.

**Super Issues**
https://github.com/tern-tools/tern/issues/1110

### PR DESCRIPTION
**Pull request by nishakm**
This commit adds instructions to deploy Tern as a Kubernetes Job.

Fixes #1111

Signed-off-by: Nisha K <nishak@vmware.com>

### CODE REVIEW

**Comment by nishakm** [comment]
@rnjudge you mentioned you had left a review, but I don't see any. Can you review please?

**Review by rnjudge — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -161,6 +162,59 @@ docker run --privileged -v /var/run/docker.sock:/var/run/docker.sock ternd --dri
...
+```
+
+Start minikube
+```
+$ minikube start --driver=virtualbox
+```
+
+Use minikube to build the Tern container image
+```
+$ minikube image build -t tern:2.9.1 -f Dockerfile .
```
[the remark this answers is not in the document]
Comment (nishakm) [reply README.md:182]: Maybe `tern:test`? For minikube image, you require a tag.

[the remark this answers is not in the document]
Comment (rnjudge) [reply README.md]: `tern:test` works for me. I'm not against a tag, just wanted to make it generic so the docs hold up long after release 2.9.1 :D

File: README.md [hunk 2]
```diff
@@ -161,6 +162,60 @@ docker run --privileged -v /var/run/docker.sock:/var/run/docker.sock ternd --dri
 
 You can make this change to the `docker_run.sh` script to make it easier.
 
+## Creating a Kubernetes Job<a name="k8s-deployment">
```
Comment (rnjudge) [inline README.md:165]: Ack, sorry I didn't catch this before :( `<a name="k8s-deployment">` should be `<a name="k8s-job"> to be consistent with how you defined it above.. (line 22)

**Review by rnjudge — APPROVED** [review]

**Pull request merged**